example/build-dev-image.dockerfile

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
Is there any other more suitable storage location?

Reply:
we may can move to [`apisix-docker`](https://github.com/apache/apisix-docker), but I don't think it's very appropriate and it will be more complicated.